src/compass/schemas/s1_cslc_geo.yaml

```diff
@@ -61,28 +63,28 @@ processing_options:
    rdr2geo: include('rdr2geo_options', required=False)
 
 geocoding_options:
-   # Format of output file
-   output_format: enum('ENVI', 'GTiff', 'COG', required=False)
-   # Boolean flag to enable/disable flattenin
-   flatten: bool(required=False)
-   # Number of lines to process in batch
-   lines_per_block: int(min=1, required=False)
-   # Product posting along X (same units as burst center EPSG)
-   x_posting: num(min=0, required=False)
-   # Product posting along Y (same units as burst center EPSG)
-   y_posting: num(min=0, required=False)
-   # Controls the product grid along X (same units as burst center EPSG)
-   x_snap: num(min=0, required=False)
-   # Controls the product grid along Y (same units as burst center EPSG)
-   y_snap: num(min=0, required=False)
+    # Format of output file
+    output_format: enum('ENVI', 'GTiff', 'COG', required=False)
```

Comment:
Do we still support this? (this is unrelated to this PR but if it is unsupported we should just get rid of it)

Reply:
We use this in s1_geocode_metadata [here](https://github.com/opera-adt/COMPASS/blob/a0a16a82279fc1d94b9f8ef9d3ac3599237e3f96/src/compass/s1_geocode_metadata.py#L47). I think it's ok to leave it. What do you think @vbrancat ?

Reply:
Nevermind, I forgot we used it there.